### ISSUE TITLE
Add SHT31 and E-Paper support

### ISSUE THREAD

**Issue by knopserl** [1technophile/OpenMQTTGateway#188]
I was thinking to use this project for a quality temperature/humidity sensor with e-paper (Wavershare, 1,54" 200x200 pixel) with LiPo battery.
I would need some help to add support for a SHT31 temp/humitidy sensor and also for a waveshare (or any other) e-paper/e-ink. The libs are available but if somebody could proper integrate it into the project, that would be great.

**Comment by BertrumUK** [comment]
Could this be added as SHT3x to include other versions of this sensor?

**Comment by 1technophile** [comment]
yes of course ;-)

**Comment by 1technophile** [comment]
great, could you submit a pull request so as to enable me to add it easily?

**Comment by 1technophile** [comment]
@stripeyhorse do you need some help so as to submit a pull request?

**Comment by stripeyhorse** [comment]
@1technophile Yes please. i did try and do it but ive ended up with https://github.com/stripeyhorse/OpenMQTTGateway

My SHT3x sensor on my m5 stack has been working great all week. I think my CCS811 is faulty thou so ive stopping using it. I have ordered more and a few different sensors as well. The code is pretty hacked together but i did try and follow the layout and style already used.

**Comment by 1technophile** [comment]
@stripeyhorse 
When on your [OMG repository main page](https://github.com/stripeyhorse/OpenMQTTGateway) you should see a button "New pull request".
Click on it
You will see the changes you have made.
Check these changes.
If ok click on "create pull request"
Name it and put some descriptions

Once done I will review the changes made

**Comment by insane2subro** [comment]
In the mean time how about adding support for SSD1306 oled displayes, these are pretty common.

**Comment by 1technophile** [comment]
@insane2subro feel free to submit a pull request also :-)
